backend/app/api/endpoints/generate_recommendation.py

```diff
@@ -0,0 +1,20 @@
+from fastapi import APIRouter
+
+import app.use_case.openai_endpoint as openai_endpoint
```

Comment:
`openai_endpoint` -> `openai_use_case`?

Reply:
How about recommendation.py? I think being in use_case folder is already an use case 😅

Reply:
You mean filename as `import app.use_case.recommendation as recommendation_use_case` right? 
I think it's pretty fine.